### PR TITLE
Fix for when there is no 'web' guard

### PR DESCRIPTION
**Pull request by JayBizzle**
If you have an application with no web guard, then the `normalizeCauser()` will not be able to get the User.

### CODE REVIEW

File: src/ActivityLogger.php [hunk 1]
```diff
@@ -30,18 +30,21 @@ class ActivityLogger
     /** @var \Illuminate\Support\Collection */
     protected $properties;
 
+    /** @var string */
+    protected $authDriver;
```
Comment (freekmurze) [inline src/ActivityLogger.php]: Why did you add this instance variable?

File: tests/ActivityloggerTest.php [hunk 2]
```diff
@@ -100,6 +100,26 @@ public function it_can_log_an_activity_with_a_causer()
     }
 
     /** @test */
+    public function it_can_log_an_activity_with_a_causer_when_there_is_no_web_guard()
+    {
+        config(['auth.guards.web' => null]);
+        config(['auth.guards.foo.driver' => 'session']);
```
Comment (freekmurze) [inline tests/ActivityloggerTest.php]: Change this to

```php
config(['auth.guards.foo' =>['driver => 'session', 'provider' => 'users']);
```

File: tests/ActivityloggerTest.php [hunk 3]
```diff
@@ -100,6 +100,26 @@ public function it_can_log_an_activity_with_a_causer()
...
+        config(['auth.guards.foo.provider' => 'users']);
```
Comment (freekmurze) [inline tests/ActivityloggerTest.php]: Remove this line

File: src/ActivityLogger.php (hunk 1 above)
Comment (JayBizzle) [inline src/ActivityLogger.php]: Because we now assign the `$authDriver` on the instance so we can use it in multiple methods.  Keep or remove?

Comment (freekmurze) [inline src/ActivityLogger.php]: Of course. Failed to see in the GitHub code review that the changes were in two seperate functions. My bad.

**Comment by freekmurze** [comment]
Thanks!
